### DISCUSSION TITLE
How To Download Successfully

### DISCUSSION THREAD

**Discussion by Totallyposh** [MalloyDelacroix/DownloaderForReddit#363] [Q&A]
Hi,

I've been trying to download full user content with no luck.  I am signed ibnto Reddit.  I have tried (3) different users and still "0" download count and it only runs for 3secs.  So it must be a setting I have wrong...I guess.  The Settings I have are essentially "0" for POST Limit; "Greater Than" for SCORE LIMIT EVAL.  LIMIT BY DATE is unchecked; NSFW is "include".  The COMMENT section is "Do Not Download" for all options.  I have Download Images, Videos, NSFW checked in the DISPLAY settings.

What am I doing wrong?  

PS: Thanks in advance

![image](https://github.com/MalloyDelacroix/DownloaderForReddit/assets/113797211/fbc2231d-e00d-4427-9eb5-65b6d03d4f1a)

![image](https://github.com/MalloyDelacroix/DownloaderForReddit/assets/113797211/91a6a747-e8b3-40a9-a464-b14b465e17f6)

![image](https://github.com/MalloyDelacroix/DownloaderForReddit/assets/113797211/a7593c86-f6d5-4c2d-9d14-c8525c398b59)

**Comment by MalloyDelacroix** [comment]
You are likely not doing anything wrong.  The app has not been working correctly since reddit implemented their new API changes at the beginning of July.  There are major problems with rate limits.  Even though the app is registered and each user authenticates through their account to use the app, we do not have access to the rate limits that reddit claimed we would have.  I am hoping this is something they are working on, but there's nothing I can do until they get it resolved.
